components/server/src/data/example-reports/example-report-hq.json

```diff
@@ -1,144 +1,315 @@
 {
+    "_id": "6049fa3b2df30500f9702a69",
     "title": "HQ",
     "report_uuid": "example-report-hq",
-    "subjects": [
-        {
-            "name": "HQ",
+    "subjects": {
+        "dc8c9e76-123f-4acc-a30b-f0b02753e908": {
             "type": "software",
-            "metrics": [
-                {
+            "name": "HQ",
```

Comment:
We should really replace this example with something else because HQ doesn't exist in Sonarcloud.io anymore.

Reply:
Maybe simply leave away this example report? I think 3 example reports should principally be sufficient, especially if we are gonna have to spend time to maintain them.